.github/workflows/check_license.yaml

```diff
@@ -0,0 +1,38 @@
+name: spdx
+
+on:
+  pull_request
+
+jobs:
+  check-spdx-headers:
+    runs-on: ubuntu-latest
+    steps:
+    
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Get changed files
+      run: |
+        git remote add boschglobal https://github.com/boschglobal/kuksa.val.git
+        git remote add eclipse https://github.com/eclipse/kuksa.val
+        git fetch origin
+        git fetch boschglobal
+        git fetch eclipse
+        if git rev-parse --verify origin/${{ github.event.pull_request.head.ref }} >/dev/null 2>&1; then
+          echo "files=$(git diff --name-only eclipse/${{ github.event.pull_request.base.ref }}..origin/${{ github.event.pull_request.head.ref }} | tr '\n' ',')" >> $GITHUB_ENV
+        elif git rev-parse --verify eclipse/${{ github.event.pull_request.head.ref }} >/dev/null 2>&1; then
+          echo "Failed to run git diff command with origin. Trying with eclipse repo..."
+          echo "files=$(git diff --name-only eclipse/${{ github.event.pull_request.base.ref }}..origin/${{ github.event.pull_request.head.ref }} | tr '\n' ',')" >> $GITHUB_ENV
+        elif git rev-parse --verify boschglobal/${{ github.event.pull_request.head.ref }} >/dev/null 2>&1; then
```

Comment:
Do we know when/if when we end up here, i.e. need to compare with boschglobal? If we could avoid referencing boschglobal it would be great.

Reply:
So, I did not find out why it is not working with origin. Then I tried boschglobal and it worked. That’s why I added eclipse and boschglobal checks. Maybe you know more? What is my understanding is that the checkout part should check out the branch but maybe the beach does not belong to a repository or it could not get resolved. What would be possible is to check via boschglobal and eclipse commits. But then other „own” forks would not work. I will do some investigating again

Reply:
Okay, I think I found a solution so resolving this :) hope you are happy now :)